4-css-grid/styles.css

```
@@ -0,0 +1,32 @@
+.content-grid {
+    display: grid;
+    grid-template: 1fr 1fr / 1fr 1fr 1fr;
+    grid-gap: 1.5rem;
+    margin-top: 1em;
+    margin-right: 5em;
+}
+
+.styles-article {
+    background-color: rgb(95, 94, 94);
+    border-radius: 0.3em;
```

```suggestion
    border-radius: .3em;
```
En aplicaciones grandes y complejas, salvar 1 caracter en CSS puede reducir mucho el peso del CSS a descargar.